openfermioncirq/utils/qubitoperator_translator.py

```diff
@@ -0,0 +1,78 @@
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+
+"""OpenFermion to CIRQ translation functions."""
+
+import cirq
+from openfermion import QubitOperator
+
+
+def qubitoperator_to_pauli_string(qubit_op):
```

Comment:
Since QubitOperators correspond conceptually to PauliSums rather than PauliString, we shouldn't expose this as a public method. Make it private.

Reply:
I fixed it by adding an underscore and removing the import from init.